README.md

```diff
@@ -1,11 +1,46 @@
 # SwapScope
 
-Welcome to **SwapScope**, a monorepo that houses the SwapScope project. This repository is composed of two distinct components: the **DApp** (Decentralized Application) and the **Publisher**. These components are organized into separate folders for your convenience.
+Swapscope is an open-source project that simplifies the process of streaming real-time Ethereum Uniswap (for now) liquidity operations (additions and removals) data. The project consists of two main components: the DApp (frontend) and the publisher (backend). 
+Swapscope's publisher utilizes [Syntropy Data Layer's (DL)](https://www.syntropynet.com/post/presenting-the-new-vision) Ethereum event log stream, processes the data, and publishes it back to Syntropy DL which frontend also utilizes to display the processed data.
 
-## Project Overview
+## Getting Started
 
-TBD
+### Prerequisites
+Before you start using (modifying) Swapscope locally, there are some prerequisites you need to fulfill:
+* Access to Syntropy Data Layer and it's streams - [Developer Portal](https://developer-portal.syntropynet.com/)
+* Go (Golang 1.20) - [Install Go](https://go.dev/doc/install)
+* Node.js and npm - [Install Node.js and npm](https://nodejs.org/en)
+* Access to Coingecko API (for token and pricing information (for now)) - [Coingecko API](https://www.coingecko.com/en/api)
+* Local database for storing token and liquidity pool information (helps to reduce API calls significantly) + publisher has built-in functionality to save liquidity addition/removal operations into a database
```

Comment:
Swapscope needs PostgreSQL specifically. It is much more convenient to guide how to run it using Docker, however, that could be covered if we added a `docker-compose.yml`.

Reply:
Same as above: https://github.com/SyntropyNet/swapscope/issues/16